irmaseal-core/src/lib.rs

```diff
@@ -31,23 +55,48 @@ pub trait Writable {
     fn write(&mut self, buf: &[u8]) -> Result<(), Error>;
```

Comment:
@rowanG077 In identity.rs you are using the write function from writable for writing to the `ArrayVec`. Is there a special reason why you did this? Is it maybe related to the performance of `.expand(slice.iter().cloned())` from the library? I did not test the performance though, but I was wondering why there is a second implementation of this in `util`.

It is the only location where `Writable` is used now, so when it can be removed I can fully remove the `Writable` trait. On all other places I use `AsyncWrite` instead.

Reply:
Extend is also possible.